### DISCUSSION TITLE
Consult paddle's random number settings

### DISCUSSION THREAD

**Discussion by Ethan1X** [PaddlePaddle/PaddleSpatial#231] [BaiduKDDCup2022]
We have set the random number of paddle and others, but there is still randomness, is there any other necessary setting? Consider the reproducibility of the algorithm.

**Comment by ramber1836** [comment]
Please refer to the train.py in the test-1.zip under the path: PaddleSpatial/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/paddlepaddle/tests

**Reply by Ethan1X** [reply]
I set it up according to the file, and the result will change when the random seed is fixed.
